src/drt/test/gcd_nangate45_distributed.tcl

```diff
@@ -5,7 +5,7 @@ set server1 [exec $OR server1.tcl > results/server1.log &]
 set server2 [exec $OR server2.tcl > results/server2.log &]
 set balancer [exec $OR balancer.tcl > results/balancer.log &]
 set base [exec $OR -exit gcd_nangate45.tcl > results/base.log &]
-
+exec sleep 3
```

Comment:
Is this intended?

Reply:
yes. to slow down the leader and let the servers be spawned before sending them any work.

Reply:
We generally need a better worker startup process.  This can hold for more productization of distributed drt.